openrtb_ext/deal_tier.go

```diff
@@ -39,7 +39,9 @@ func ReadDealTiersFromImp(imp openrtb2.Imp) (DealTierBidderMap, error) {
 	}
 	for bidder, param := range impPrebidExt.Prebid.Bidders {
 		if param.DealTier != nil {
-			dealTiers[BidderName(bidder)] = *param.DealTier
+			if bidderNormalized, bidderFound := NormalizeBidderName(bidder); bidderFound {
+				dealTiers[bidderNormalized] = *param.DealTier
+			}
```

Comment:
The deal tier map being created here from the original request has keys that are the normalized bidder names. This map is then used in `applyDealSupport` but that function does not appear to alter the bidder name in `winningBidsByBidder` --> `topBidsPerImp` prior to checking if the bidder has any deal tiers in the deal tier map. Is a change needed in `applyDealSupport` as well?

Reply:
Good catch. Modified.